### PR TITLE
Update documents to reflect ACT format latest draft

### PR DESCRIPTION
**Pull request by WilcoFiers**

### CODE REVIEW

File: pages/design/rule-design.md [hunk 1]
```diff
@@ -2,126 +2,71 @@
...
 
 3. **Background**: A list of resources that support the workings of the rule.
 
 4. **[Assumptions](#assumptions)**: Explicitly state all assumptions made by the rule to ensure accountability of the results.
 
-5. **[Rule properties](#rule-properties)**: Define the test subject and its environment, as well as other meta data.
+5. **[Applicability](#applicability)**: Identify which elements on a page (if any) are to be tested using the rule.
 
-6. **[Selector](#selector)**: Use selectors to group the tests by page element and Success Criterion.
+6. **[Expectations](#expectations)**: Assert everything what must be true about the target elements, in order for them to pass the rule.
```
Comment (carlosapaduarte) [inline pages/design/rule-design.md]: Assert everything **that** must be true
or
Assert what must be true

File: pages/design/rule-design.md [hunk 2]
```diff
@@ -2,126 +2,71 @@
...
-- **DOM and CSS**: Same as 2. plus computed style for each element)=
-- **Rendered page**: Page as it is presented in a browser
-- **Rendered page + server connection**: Page as it is presented in a browser, as well as an open connection (and any cookies or other session data that might be required)
+- HTTP Messages: All messages sent through HTTP(S)
+- DOM Tree: The tree that HTML is parsed into
+- CSS Styling: CSS applied to lay out and style the DOM Tree
+- Accessibility Tree: The tree that user agents expose to the accessibility API
 
-**Note**: The rule should specify the minimum level of pre-processing. A test that is carried out on the DOM can usually also be carried out on the rendered page but the the latter needs more processing power. It should also be kept in mind that the use of a (headless) browser can introduce bugs in the test procedure.
+Other aspects may be necessary for testing. These can be added as well as long as they are sufficiently defined.
```
Comment (carlosapaduarte) [inline pages/design/rule-design.md]: Remove "as well"

File: pages/design/rule-design.md [hunk 3]
```diff
@@ -2,126 +2,71 @@
...
+## Expectations
 
-## Test steps
+The applicability help the testers (or test tools) identify what has to be checked. Following that, the expectations are statements that must be true for the applicable elements to pass the rule. If any of the expectations is false, than the target element failed the rule. 
 
-The selectors help the user (or tool) identify what has to be checked. The goal of the Auto-WCAG test design is to cover the complete workflow, i.e. all steps / all tests that are necessary to reach a conclusion. The procedure contains automated and manual steps. Usually a combination of both.
+Each expectation exposes a reason why an element may not meet a particular conformance requirement. The expectations can be "linked", in that one has to be met before a second can be tested. For example, a rule testing link names may have as its first expectation "Target element has an accessible name", and as a second expectation "Expectation 1 is true for the target element, and the accessible name is describes the function of the target element".
```
Comment (carlosapaduarte) [inline pages/design/rule-design.md]: Remove the last "is" in the final sentence

File: pages/structure/aggregation.md [hunk 4]
```diff
@@ -2,20 +2,19 @@
 title: Result Aggregation
 ---
 
-Each test case of the Auto-WCAG group is related to a specific WCAG 2.0 success criterion. A success criterion can (and usually does) have multiple test cases associated with it. Each test case can have one of three outcomes: Pass, CannotTell and Fail. The results of individual test cases can be combined in the following manner, to find the result of a specific success criterion.
+Each rule of the Auto-WCAG group is related to a specific WCAG 2.x success criterion. A success criterion can (and usually does) have multiple rules associated with it. Each rule can have one of three outcomes: Pass, CannotTell and Fail. The results of individual rule can be combined in the following manner, to find the result of a specific success criterion.
```
Comment (carlosapaduarte) [inline pages/structure/aggregation.md]: The last rule should be rules

**Comment by WilcoFiers** [comment]
List of changes to make:

- Put acc req after description in examples
- background after implementation tests
- Template needs an Issues section - Maybe dynamic to Github issues with tags
- Implementation Tests -> Test Cases
- Remove -> environment: Markup Document / DOM Structure / Web Browser / WebDriver
- Success_criterion -> accessibility_requirements
- Add "language" to test aspects
- accessibility_requirements below Description

**Pull request merged**